cucumber/src/main/java/foundation/stack/datamill/cucumber/CommandLineSteps.java

```diff
@@ -267,24 +267,19 @@ public void cleanUp() throws IOException {
         BufferedReader processOutput = new BufferedReader(new InputStreamReader(inputStream));
         List<String> output = new CopyOnWriteArrayList<>();
 
-        Thread processOutputThread = new Thread(() -> {
+        String line = null;
```

Comment:
Removing the separate thread here is risky - using a thread pool would be a better alternative.